### PR TITLE
Update the_big_picture.rst

### PR DESCRIPTION
**Pull request by oldskool**
When following the steps in this documentation, I ended up with this: http://i.imgur.com/nK3XM2B.png
Got the suggestion to install the assets on the IRC channel and it did the trick for me. This addition to the docs might help others facing the same problem to make their app look as expected.

### CODE REVIEW

**Comment by xabbuh** [comment]
@oldskool Actually I think we should better change the URL to be `http://localhost:8000/app_dev.php/` instead. Otherwise, you won't see the same as what is shown in the screenshot (for example, the web debug toolbar is not present in the prod environment).

**Comment by WouterJ** [comment]
@xabbuh the built-in server is used, which will always be in dev env unless you use `--env=prod`

**Comment by xabbuh** [comment]
@WouterJ Hm, yes you're right. But why should it be needed at all then to manually install assets?

**Comment by WouterJ** [comment]
@xabbuh I need to do it quite often too. Most of the time, because the composer scripts fail at clearing the cache on installation, the script that installs the assets is then no longer executed.

**Comment by oldskool** [comment]
@xabbuh I tried that as well, but that didn't render the assets either. In this part of the docs, the PHP built-in webserver is still used, so it seems that @WouterJ is right about the dev environment already being used.

**Comment by xabbuh** [comment]
Any idea why this is failing on some systems?

**Comment by oldskool** [comment]
No clue, running this PHP version if it's of any help:

> $ php -v
PHP 5.5.11 (cli) (built: Apr  8 2014 15:07:14)
Copyright (c) 1997-2014 The PHP Group
Zend Engine v2.5.0, Copyright (c) 1998-2014 Zend Technologies
    with Xdebug v2.2.3, Copyright (c) 2002-2013, by Derick Rethans

**Comment by xabbuh** [comment]
Then let's merge this imo. It does not hurt and it may help people.

@oldskool Can you please rebase your PR?

**Comment by oldskool** [comment]
@xabbuh Done.

**Comment by xabbuh** [comment]
:+1:

File: quick_tour/the_big_picture.rst [hunk 1]
```diff
@@ -106,6 +106,13 @@ Congratulations! Your first Symfony project is up and running!
     them are explained in the
     :ref:`Setting up Permissions <book-installation-permissions>` section
     of the official book.
+    
+    If the welcome page does not seem to be rendering CSS or image assets,
+    install them first:
+    
+    .. code-block:: bash
+    
+        $ php app/console assets:install web
```
Comment (timglabisch) [inline quick_tour/the_big_picture.rst]: shouldn't we remove the "web" parameter?

Comment (WouterJ) [inline quick_tour/the_big_picture.rst]: Hmm, I think we should. @oldskool can you please add this change? I promise that we'll merge it after you've done that :)

**Comment by oldskool** [comment]
@WouterJ Done.

**Comment by WouterJ** [comment]
Thanks for the addition and your patience, Jan! It's merged into the 2.3 docs now.

**Pull request closed without merging**